scripts/basic-workflow-tests.sh

```diff
@@ -82,19 +86,8 @@ test_serve() {
     fi
 
     task Serve the model
-    ilab model serve "${SERVE_ARGS[@]}" &
-
-    ret=1
-    for i in $(seq 1 10); do
-        sleep 5
-    	step "$i"/10: Waiting for model to start
-        if curl -sS http://localhost:8000/docs > /dev/null; then
-            ret=0
-            break
-        fi
-    done
-
-    return $ret
+    ilab model serve "${SERVE_ARGS[@]}" &> serve.log &
```

Comment:
Do we need something above that sets the `--model-path` in `SERVE_ARGS` to a model vLLM can use? If the `-v` flag is set I'm not sure what the model path will be here and how ilab serve will know to use vLLM as the backend for it.

Reply:
That's correct, let me fix that. Thanks!